src/modules/reserve-overview/components/ReserveInformation/index.tsx

```diff
@@ -213,22 +208,12 @@ export default function ReserveInformation({
               />
             </APYCard> */}
 
-            <APYCard title={intl.formatMessage(messages.variableBorrowing)} color="secondary">
+            <APYCard title={intl.formatMessage(messages.borrowing)} color="secondary">
               <APYLine
                 title={intl.formatMessage(messages.borrowAPY)}
                 value={reserveOverviewData.variableAPY}
                 condition={reserveOverviewData.borrowingEnabled}
               />
-              <APYLine
```

Comment:
I suggest we uncomment instead of remove the code unless really necessary. This also applies to other changes.